architecture/index.adoc

```diff
@@ -10,13 +10,25 @@ toc::[]
 The foundation of {product-title} is based on Kubernetes and therefore shares the same technology.
 To learn more about {product-title} and Kubernetes, see xref:../architecture/architecture.adoc#architecture[product architecture].
 
+include::modules/openshift-architecture-common-terms.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information on networking, see xref:../networking/understanding-networking.adoc#understanding-networking[{product-title} networking].
+* For more information on storage, see xref:../storage/index.adoc#index[{product-title} storage].
+* For more information on authentication, see xref:../authentication/index.adoc#index[{product-title} authentication].
+* For more information on Operator Lifecycle Manager (OLM), see xref:../operators/understanding/olm/olm-understanding-olm.adoc#olm-understanding-olm[OLM].
+* For more information on logging, see xref:../logging/viewing-resource-logs.adoc#viewing-resource-logs[{product-title} Logging].
+* For more information on Over-the-air (OTA) updates, see xref:../updating/index.adoc#index[Updating {product-title} clusters].
```

Comment:
Per Andrea's comment about another instance of the O in over-the-air needing to be lowercase

Reply:
Addressed